deploy/environments/applications.yaml

```diff
@@ -63,6 +63,7 @@ spack:
     - py-pytouchreader
     - py-simwriter
     - py-sonata-network-reduction%gcc ^neuron%intel^py-ipython%gcc
+    - py-rtree
```

Comment:
Is this a leftover that should be removed because it's now in libraries?

Reply:
Yes, indeed, I missed that. Thanks. To be addressed in the next commit.